src/rust/engine/graph/src/lib.rs

```diff
@@ -487,6 +508,7 @@ impl<N: Node> InnerGraph<N> {
     self.nodes.get(node)
   }
 
+  // TODO: Now that we never delete Entries, we should consider making this infalliable.
```

Comment:
We're probably going to want to start deleting entries again at some point for memory reasons, just in a more principled/less aggressive way